tests/bettertransformer/test_bettertransformer_audio.py

```diff
@@ -0,0 +1,143 @@
+
+        input_dict = feature_extractor(input_features, return_tensors="pt", padding=True)
+        return input_dict
+
+    def test_logits(self):
+        r"""
+        This tests if the converted model produces the same logits
+        than the original model.
+        """
+        # The first row of the attention mask needs to be all ones -> check: https://github.com/pytorch/pytorch/blob/19171a21ee8a9cc1a811ac46d3abd975f0b6fc3b/test/test_nn.py#L5283
+
+        for model_to_test in self.all_models_to_test:
+            inputs = self.prepare_inputs_for_class(model_to_test)
+
+            torch.manual_seed(0)
+            hf_random_model = AutoModel.from_pretrained(model_to_test).eval()
+            random_config = hf_random_model.config
+
+            torch.manual_seed(0)
+            converted_model = BetterTransformer.transform(hf_random_model)
+
+            torch.manual_seed(0)
+            hf_random_model = AutoModel.from_pretrained(model_to_test).eval()
+            random_config = hf_random_model.config
+
+            self.assertFalse(
+                hasattr(hf_random_model, "use_bettertransformer"),
+                f"The model {hf_random_model.__class__.__name__} has been converted to a `fast` model by mistake.",
+            )
+
+            with torch.no_grad():
+                r"""
+                Make sure the models are in eval mode! Make also sure that the original model
+                has not been converted to a fast model. The check is done above.
+                """
+                torch.manual_seed(0)
+                hf_hidden_states = hf_random_model(**inputs)[0]
+
+                torch.manual_seed(0)
+                bt_hidden_states = converted_model(**inputs)[0]
+
+                if "gelu_new" in vars(random_config).values():
+                    # Since `gelu_new` is a slightly modified version of `GeLU` we expect a small
+                    # discrepency.
+                    tol = 4e-2
+                else:
+                    tol = 1e-3
+
+                self.assertTrue(
+                    torch.allclose(bt_hidden_states[0][-3:], torch.zeros_like(bt_hidden_states[0][-3:])),
+                    "The BetterTransformers converted model does not give null hidden states on padded tokens",
+                )
+
+                self.assertTrue(
+                    torch.allclose(hf_hidden_states[-1], bt_hidden_states[-1], atol=tol),
+                    "The BetterTransformers Converted model does not produce the same logits as the original model. Failed for the model {}".format(
+                        hf_random_model.__class__.__name__
+                    ),
+                )
+
+    def test_raise_autocast(self):
+        r"""
+        Autocast seems to be incompatible with `Wav2Vec2` and `cpu`, let's skip this
+        test. The test below should be fine to test if the `forward_checker` has been run correctly.
+        """
+        pass
```

Comment:
Why?

Reply:
I missed somthing there, I think that it should work now

Reply:
Fixed in [a958177](https://github.com/huggingface/optimum/pull/470/commits/a958177cb8f193ac54e8174fab997a31b85d3750)